geomstats/backend/numpy.py

```diff
@@ -8,10 +8,6 @@
 float64 = np.float64
 
 
-def flip(*args, **kwargs):
```

Comment:
Can you rebase on the latest version of master? This function has been added in a recent pull request.

https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

Reply:
(it will solve the unit test failure)

Reply:
It took me a while to figure it out but it's done ! Hope it worked.